scripts/utils.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+
+// eslint-disable-next-line import/no-nodejs-modules
+import fs from "fs";
+
+type Package = {|
+  name: string,
+  version: string,
+|};
+
+type NodeOps = {|
+  web: {|
+    staticNamespace: string,
+  |},
+|};
+
+export function getPackage(): Package {
+  if (!fs.existsSync("./package.json")) {
+    throw new Error(`Package file not found`);
+  }
+  return JSON.parse(fs.readFileSync("./package.json", "utf-8"));
+}
+
+export function getNodeOps(): NodeOps {
```

Comment:
Do we use this `getNodeOps()` function anywhere? If not, can we delete it?

Reply:
Ah right.  Should we use it to update the namespace in the CDN URL, like we do with package version?  I like the idea of making it dynamic but it's another point of failure as well.

Reply:
I think the .nodeops file can remain static similar to what we do in paypal/sdk-release.

Reply:
I thought it might be confusing if somebody ever tried to change the namespace and the urls didn't update.  I don't see why we would ever need to change the namespace, though

Reply:
oh I see your point. We can keep it then to validate the namespace name since its directly tied to the cdn url.